.circleci/config.yml

```diff
@@ -1,45 +1,18 @@
-version: 2
+version: 2.1
 jobs:
-  build:
-    docker:
-      - image: circleci/python:3.7.3
-    steps:
-    - checkout
-    - run: pwd
-    - restore_cache:
-       keys:
-         - requirements-{{ checksum "requirements.txt" }}
-         - requirements- # used if checksum fails
-    - run: python3 -m venv venv
-    - run: source venv/bin/activate
-    - run: pip install --user -r requirements.txt
-    - save_cache:
-        key: requirements-{{ checksum "requirements.txt" }}
-        paths:
-          - venv
-    - persist_to_workspace:
-        root: /home/circleci
-        paths:
-          - project
   test:
     docker:
-      - image: circleci/python:3.7.3
+      - image: cimg/python:3.8.6
     steps:
-      - attach_workspace:
-          at: /home/circleci
-      - run: pwd
-      - run: source venv/bin/activate
+      - checkout
+      - run: pipenv install -r requirements.txt
```

Comment:
If using pipenv, I recommend switching to a pipfile https://pipenv-fork.readthedocs.io/en/latest/basics.html
This will work with the orb and cache your deps